src/utils/api.js

```diff
@@ -2,8 +2,10 @@ import axios from "axios";
 
 const ncNewsApi = axios.create({ baseURL: "https://nc-news-5aio.onrender.com/api" })
 
-export const getArticles = () => {
-    return ncNewsApi.get(`/articles`)
+export const getArticles = (topic) => {
+    let queryString = "/articles"
+    if (topic) queryString+= `?topic=${topic}`
```

Comment:
This is nice, although there's a slightly nicer way to do it using axios params. Checkout the docs for clues!

Reply:
Thanks Rose :)